openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx

```diff
@@ -194,22 +187,22 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
               {t('Generate an article')}
             </MenuItem>
           )}
-          <MenuItem onClick={() => handleAskAi('spelling')}>
+          <MenuItem onClick={() => handleAskAi('spelling')} disabled={messageInput.length === 0}>
```

Comment:
I'm unsure we should make a test on messageInput. My understanding is that messageInput is the input field of the popover when generating a new article. So here, we can't have this button active until we open the generate an article popover and type something in the Input field. I think we want to be able to check the spelling of what's inside the content field so the correct test should be currentValue.length === 0, don't you think ?

Reply:
Yes! I'll change it!

Reply:
In the code: messageInput is initialize with 'current value':   
const [messageInput, setMessageInput] = useState<string>(currentValue);